pandas/tools/pivot.py

```diff
@@ -75,6 +75,11 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     Returns
     -------
     table : DataFrame
+
+    See also
+    --------
+    DataFrame.pivot : pivot without aggregation that can handle
+        non-numeric data
```

Comment:
Ah, thank you for bringing that to my attention. I'll push that change later tonight.